File: backend/src/main/java/com/wgzhao/addax/admin/AdminApplication.java
package com.wgzhao.addax.admin;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableAsync
@EnableScheduling
public class AdminApplication
{
    public static void main(String[] args)
    {
        SpringApplication.run(AdminApplication.class, args);
    }
}

File: src/main/kotlin/com/wgzhao/addax/admin/AdminApplication.kt
package com.wgzhao.addax.admin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableAsync
@EnableScheduling
object AdminApplication {
    fun main(args: Array<String>) {
        runApplication<AdminApplication>(*args)
    }
}
